src/snowflake/snowpark/functions.py

```diff
@@ -666,45 +672,99 @@ def mode(e: ColumnOrName) -> Column:
 
 def skew(e: ColumnOrName) -> Column:
     """Returns the sample skewness of non-NULL records. If all records inside a group
-    are NULL, the function returns NULL."""
+    are NULL, the function returns NULL.
+
+    Example::
+        >>> df = session.create_dataframe(
+        ...     [10, 10, 20, 25, 30],
+        ...     schema=["a"]
+        ... ).select(skew("a"))
+        >>> df.collect()
+        [Row(SKEW("A")=0.05240788515)]
```

Comment:
can we lower the precision upto 3-4 decimal places? some of the earlier added examples have done so